src/mme/metrics.h

```diff
@@ -0,0 +1,42 @@
+#ifndef MME_METRICS_H
+#define MME_METRICS_H
+
+#include "ogs-metrics.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum mme_metric_type_global_s {
+    MME_METR_GLOB_GAUGE_ENB_UE,
+    MME_METR_GLOB_GAUGE_MME_SESS,
+    MME_METR_GLOB_GAUGE_ENB,
+    _MME_METR_GLOB_MAX,
+} mme_metric_type_global_t;
+extern ogs_metrics_inst_t *mme_metrics_inst_global[_MME_METR_GLOB_MAX];
+
+static inline void mme_metrics_inst_global_set(mme_metric_type_global_t t, int val)
+{
```

Comment:
I updated all of these to be "{ gs_metrics_inst_set(mme_metrics_inst_global[t], val); }" in one line.
Now we have only one node/section, but once we start having more this will take a lot of lines.